src/main/java/com/kuba6000/mobsinfo/api/MobDrop.java

```diff
@@ -75,8 +75,8 @@ public MobDrop(ItemStack stack, DropType type, int chance, Integer enchantable,
         this.playerOnly = playerOnly;
     }
 
+    @SuppressWarnings("unchecked")
     public MobDrop copy() {
-        @SuppressWarnings("unchecked")
         MobDrop copy = new MobDrop(
```

Comment:
If you are moving the suppresion to a whole method, then remove the `// noinspection unchecked` below